src/resources/locales/en.json

```diff
@@ -854,9 +854,229 @@
+                },
+                "NavigatePreviousChapterAlt": {
+                    "description": "in reader window, move to previous \"chapter\" (spine item in the reading order of the publication). Note that the keyboard focus is automatically moved to the \"main\" area landmark, which is immediately before the frame that displays publication content (just like when following / activating a link in the table of contents).",
+                    "name": "Go to Previous Chapter (Alternative)"
+                },
+                "NavigatePreviousHistory": {
+                    "description": "in reader window, go back or forward in the navigation history: Beginning or End of the publication.",
+                    "name": "Go to Previous History"
+                },
+                "NavigatePreviousLibraryPage": {
+                    "description": "In the library view, display the previous set of books.",
+                    "name": "Go to Previous Library Page"
+                },
+                "NavigatePreviousLibraryPageAlt": {
+                    "description": "In the library view, display the previous set of books (alternative keyboard shortcut).",
+                    "name": "Go to Previous Library Page (Alternative)"
+                },
+                "NavigatePreviousOPDSPage": {
+                    "description": "in library window for Catalogues (OPD) feeds, activates the previous pagination links.",
+                    "name": "Go to Previous OPDS Page"
+                },
+                "NavigatePreviousOPDSPageAlt": {
+                    "description": "in library window for Catalogues (OPD) feeds, activates the previous pagination links (alternative keyboard shortcut).",
+                    "name": "Go to Previous OPDS Page (Alternative)"
+                },
+                "NavigatePreviousPage": {
+                    "description": "in reader window, move to previous or next \"page\" (or scroll unit, if pagination is disabled).",
+                    "name": "Go to Previous Page"
+                },
+                "NavigatePreviousPageAlt": {
+                    "description": "in reader window, move to previous or next \"page\" (or scroll unit, if pagination is disabled).",
+                    "name": "Go to Previous Page (Alternative)"
+                },
+                "NavigateToBegin": {
+                    "description": "in reader window, goes to the beginning of the publication.",
+                    "name": "Go to Beginning"
+                },
+                "NavigateToEnd": {
+                    "description": "in reader window, goes to the end of the publication.",
+                    "name": "Go to End"
+                },
+                "OpenReaderInfo": {
+                    "description": "in reader window, publication info popup modal dialog (may be used in future Thorium version for similar feature in library window).",
```

Comment:
`may be used in future Thorium version for similar feature in library window` wtf !? :)